components/script/dom/htmlmediaelement.rs

```diff
@@ -1376,6 +1379,20 @@ impl HTMLMediaElementMethods for HTMLMediaElement {
     fn Played(&self) -> DomRoot<TimeRanges> {
         TimeRanges::new(self.global().as_window(), self.played.clone())
     }
+
+    fn GetVolume(&self) -> Fallible<Finite<f64>> {
```

Comment:
Add a comment to link to the specification: https://html.spec.whatwg.org/multipage/media.html#dom-media-volume.